### PR TITLE
Frontend

### PR DESCRIPTION
**Pull request by thisisobate**

### CODE REVIEW

**Comment by MyloCarson** [comment]
Please add image to this pull request. So we see visuals of what u did. No vx.

**Comment by thisisobate** [comment]
Okay. 
Screenshot (284)
Screenshot (286)

File: blog-detail-dark.php [hunk 1]
```diff
@@ -12,300 +12,312 @@
     <link rel="shortcut icon" href="favicon.png" type="image/x-icon">
```
Comment (MyloCarson) [inline blog-detail-dark.php:12]: Your pull request says, you add 404 pages. But you edited blog-detail-dark.php as well. Why?

**Review by MyloCarson — COMMENTED** [review]
Your pull request says, you add 404 pages. But you edited blog-detail-dark.php as well. Why?

File: blog-detail-dark.php (hunk 1 above)
Comment (thisisobate) [reply blog-detail-dark.php:12]: I resolved a merge conflict bro. Please understand

Comment (MyloCarson) [reply blog-detail-dark.php:12]: Okay. no vex. I am just trying to make sure things are not changed. U get ? thanks man for ur understanding

Comment (thisisobate) [reply blog-detail-dark.php:12]: No wahala bro. I get

**Pull request merged**